build/tools/Automation/Services/IHttpService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Oryx.Automation.Services
+{
+    public interface IHttpService
+    {
+        public Task<string> GetDataAsync(string url);
```

Comment:
nit: the `public` access modifier isn't needed for interface methods (prior to C# 8.0, it would've caused a compilation error) since it's the default value when no access modifier is presented, but it's OK to keep this as-is (unless there are other changes that need to be made, then it would be a nice clean up change to make)

Reply:
Updated. I might have copied the signature from the `HttpService`